imap_data_access/cli.py

```diff
@@ -147,18 +148,32 @@ def main():
     )
     download_help = (
         "Download a file from the IMAP SDC to the locally configured data directory. "
+        "Run 'download -h' for more information. "
+    )
+    help_menu_download = (
```

Comment:
```suggestion
    download_description = (
```

Since these strings (this one, `help_menu_query`, and `help_menu_upload`) are being used as descriptions in the `descriptions` parameter later on in the code, perhaps this is a better variable name? No strong opinions on this one.

Reply:
I changed the name to be more specific as to where the statement occurs in the cli.